DUI3-DX/Converters/Revit/Speckle.Converters.RevitShared/Services/TransactionManagementService.cs

```diff
@@ -0,0 +1,249 @@
+using Autodesk.Revit.DB;
+using Speckle.Converters.RevitShared.Helpers;
+using Speckle.Core.Logging;
+
+namespace Speckle.Converters.RevitShared.Services;
+
+/// <summary>
+/// Is responsible for all functionality regarding subtransactions, transactions, and transaction groups.
+/// This includes starting, pausing, committing, and rolling back transactions
+/// </summary>
+public sealed class TransactionManagementService : ITransactionManagementService
+{
+  private readonly Lazy<RevitConversionContextStack> _lazyContextStack;
```

Comment:
Any reason for making this lazy? Is this essentially a singleton with the context of a conversion?

Reply:
this was needed because it was being instantiated in a dependency of the binding (right when Revit was opened before there was an active document) which was causing an error because the context stack is instantiated using the active document. However, this is no longer needed after moving the uow out to the binding.